js/index.js

```diff
@@ -15,10 +15,17 @@ export default class Teamshares {
 
     // Overwrite Rails UJS's selectors to include Shoelace elements
     Rails.buttonClickSelector = {
-      selector: "button[data-remote]:not([form]), button[data-confirm]:not([form]), sl-button[data-remote]:not([form]), sl-button[data-confirm]:not([form])",
+      selector: Rails.buttonClickSelector.selector + ", sl-button[data-remote]:not([form]), sl-button[data-confirm]:not([form])",
       exclude: "form button, form sl-button",
     };
-    Rails.linkClickSelector = "a[data-confirm], a[data-method], a[data-remote]:not([disabled]), a[data-disable-with], a[data-disable], sl-button[href][data-confirm], sl-button[href][data-method], sl-button[href][data-remote]:not([disabled]), sl-button[href][data-disable-with], sl-button[href][data-disable]";
+    Rails.linkClickSelector += ", sl-button[href][data-confirm], sl-button[href][data-method], sl-button[href][data-remote]:not([disabled]), sl-button[href][data-disable-with], sl-button[href][data-disable]";
```

Comment:
Rather than copying what was in the selectors previously, I'm now just appending the Shoelace stuff.